README.md

```diff
@@ -35,6 +36,17 @@ Docker container can be cleaned up using
 docker rm -f hintr
 ```
 
+### Input data
+
+Input data should be written to `/uploads` directory in the docker container, then when requesting validation pass the absolute path to the file in the request JSON e.g.
+
+```
+{
+  "type": "pjnz",
+  "path": "uploads/Botswana.pjnz"
```

Comment:
So this would actually be `/uploads/Botswana.pjnz`. 

You're right, this app could be totally agnostic about the parent directory and expect absolute rather than relative paths; all config could live in the Kotlin app. I think the advantage of a configured upload directory here as well would be more flexibility for local dev and testing so you might end up wanting to add configurability for those reasons down the line.